ivy.xml

```diff
@@ -13,5 +13,7 @@
     <dependency org="org.openmicroscopy" name="omero-gateway" rev="${versions.omero-gateway}"/>
     <!-- runtime dependencies from dsl/ivy.xml -->
     <dependency org="janino" name="janino" rev="${versions.janino}"/>
+    <!-- Useful for globally overriding the Bio-Formats version, empty version is ignored by default -->
```

Comment:
Ah, good to know. Then I might investigate adding in omero-model etc. as well.

Reply:
@joshmoore Yeah, limited testing so far but the assumption seems to hold as far as I can see. I don't think it's explicitly declared anywhere to be the case.